panel/config.py

```diff
@@ -433,19 +437,29 @@ def __getattribute__(self, attr):
         """
         from .io.state import state
 
-        if attr == '_param__private':
-            return super().__getattribute__('_param__private')
+        # ATTRS[attr]+=1
+        # print(ATTRS)
+        if attr in ('_param__private', '__class__', 'npm_cdn', 'param'):
```

Comment:
`npm_cdn` and `param` are asked for very frequently. So I move them to the top. I don't understand if that is a problem.

Reply:
Have to think about `npm_cdn` but everything else makes sense here for sure.